pkg/ruler/ruleset.go

```diff
@@ -89,14 +89,25 @@ func NewRuleset(logger *zap.SugaredLogger) *Ruleset {
 	runAsUserRule := Rule{
 		Predicate: rules.RunAsUser,
 		ID:        "RunAsUser",
-		Selector:  "containers[] .securityContext .runAsUser -gt 10000",
+		Selector:  ".spec, .spec.containers[] | .securityContext .runAsUser -gt 10000",
 		Reason:    "Run as a high-UID user to avoid conflicts with the host's user table",
 		Kinds:     []string{"Pod", "Deployment", "StatefulSet", "DaemonSet"},
 		Points:    1,
 		Advise:    4,
 	}
 	list = append(list, runAsUserRule)
 
+	runAsGroupRule := Rule{
+		Predicate: rules.RunAsGroup,
+		ID:        "RunAsGroup",
+		Selector:  ".spec, .spec.containers[] | .securityContext .runAsGroup -gt 10000",
+		Reason:    "Run as a high-UID group to avoid conflicts with the host's user table",
```

Comment:
```suggestion
		Reason:    "Run as a high-UID group to avoid conflicts with the host's registered groups",
```

not sure the term table is used too often for linux users & groups. And since we're adding this new one it feels slightly weird to call it a "user table" when it's about groups

What do you think about this wording?

Reply:
Maybe we can just use `host's groups` and `host's users`. I can't remember seeing "registered" regarding Linux user/group management.

Reply:
So I've updated with, wdy ?:

* `Run as a high-UID user to avoid conflicts with the host's users`
* `Run as a high-UID group to avoid conflicts with the host's groups`